public/js/constants/recipeData.js

```diff
@@ -0,0 +1,737 @@
+export const cuisineList = [
+  'African',
+  'American',
+  'Asian',
+  'BBQ',
+  'British',
+  'Caribbean',
+  'Chinese',
+  'French',
+  'Greek',
+  'Indian',
+  'Irish',
+  'Italian',
+  'Japanese',
+  'Korean',
+  'Lebanese',
+  'Mediterranean',
+  'Mexican',
+  'Moroccan',
+  'Nordic',
+  'North African',
+  'Persian',
+  'Polish',
+  'Portuguese',
+  'South American',
+  'Spanish',
+  'Thai',
+  'Turkish',
+  'Vietnamese'
+];
+
+export const categoryList = [
+  'Baking',
+  'Barbecue',
+  'Breakfast',
+  'Budget',
+  'Canapes',
+  'Dessert',
+  'Dinner',
+  'Dinner party',
+  'Drinks and cocktails',
+  'Healthy eating',
+  'Lunch',
+  'Main course',
+  'Picnic',
+  'Salads',
+  'Sandwich',
+  'Sides',
+  'Soup',
+  'Snacks',
+  'Starters'
+];
+
+export const celebrationList = [
+  'Baisakhi',
+  'Christmas',
+  'Diwali',
+  'Easter',
+  'Eid',
+  'Halloween',
+  'Hanukkah',
+  'Passover',
+  'Thanksgiving'
+];
+
+export const dietaryList = [
+  'Low sugar',
+  'Low fat',
+  'High fibre',
+  'Nut free',
+  'Gluten free',
+  'Dairy free',
+  'Egg free',
+  'Vegetarian',
+  'Vegan'
+];
+
+export const chefs = [
+  'Nigel Slater and Observer readers',
```

Comment:
I agree, but was just matching the functionality of recipeasy for now. Adding contributors would be a great improvement though